app/(public)/components/patterns/horizontal-card.tsx

```diff
@@ -19,81 +23,61 @@ interface CardProps {
   }
   linkStyle?: string
   className?: string
-  backgroundSprinkles ?: string
+  backgroundSprinkles?: string
 }
 
-const HorizontalCard = ({video, image, superHeader, header, footer, body, link, linkStyle, backgroundSprinkles, ...props}: CardProps) => {
+const HorizontalCard = ({video, image, superHeader, header, footer, body, link, linkStyle, backgroundSprinkles = 'top_right', ...props}: CardProps) => {
```

Comment:
@pookmish I'm still learning, but doesn't this make `top_right` default?

Reply:
Yeah. If it should be top left, feel free to adjust.